src/partials/header-content.hbs

```diff
@@ -39,6 +39,11 @@
             <a class="button is-primary" href="#">Download</a>
           </span>
         </div>
+        <div class="navbar-item">
+          <span class="control">
+            <button class="button is-primary" id="theme-toggle"></button>
```

Comment:
This is a placeholder button until we get our new nav structure in place, I just wanted to test the toggle mechanism